pages/index.vue

```diff
@@ -208,7 +208,7 @@
 
 			if (user.user_type !== 'member') {
 				fromSponsorshipPromise = $directus.request(readItems('gp_credits_additions', {
-					filter: { github_id: { _eq: user.external_identifier || 'admin' }, comment: { _icontains: 'recurring' } },
+					filter: { github_id: { _eq: user.external_identifier || 'admin' }, comment: { _icontains: 'recurring' }, date_created: { _gte: '$NOW(-30 day)' } },
```

Comment:
Good catch, but I think the sponsorship interval could be anywhere in 28 - 31 days, depending on the month, plus maybe some small variation in case of processing issues? How about we take all records in the past 35 days, and use the most recent one as estimate here? Seems even like a good test case.